test/blackbox-tests/test-cases/ocaml-index/gh10985/run.t

```diff
@@ -0,0 +1,23 @@
+This test attempts to build the ocaml index while depending on a library
+installed through a lock file.
+
+This is based off of the pkg/libraries.t test, just taking advantage of
+being in its own directory.
```

Comment:
I thing generally we try to have issues in a single cram file unless it is too complicated. This looks fairly straightforward.

Reply:
The new commit is in a single file